fjohnson_assignment_4/classy_tiny_instacart_2.rb

```diff
@@ -0,0 +1,188 @@
+module NewUser
+
+    class << self
+       attr_accessor :name, :phone, :address
+    end
+
+    def name
+        @name
+    end
+
+    def phone
+        @phone
+    end
+    
+    def address
+        @address
+    end
+
+    def create_account
+        puts "Welcome to Instacart!"
+        print "\nWhat is your name? "
+        @name = gets.chomp
+        print "Hi, #{@name}! What is your phone number? "
+        @phone = gets.chomp
+        print "What is your address? "
+        @address = gets.chomp
+        puts "\nThank you! Your account has been created."
+        puts "\n"
+        puts "_" * 50
+        puts "\tName: #{@name}"
+        puts "\tPhone Number: #{@phone}"
+        puts "\tAddress: #{@address}"
+        puts "-" * 50
+    end
+end
+
+module GroceryList
+
+    class << self
+        attr_accessor :item_name, :item_brand, :item_quantity, :item_instructions
+    end
+
+    def item_name
```

Comment:
When you use `attr_accessor` these should be unnecessary, just don't wrap it in `class << self`